src/www/ui/admin-dashboard-general.php

```diff
@@ -242,6 +243,12 @@ function DatabaseQueries()
         $V .= "<td class='dashboard'>$StartTime</td>";
         $V .= "<td class='dashboard'>$row[elapsed]</td>";
         $V .= "</tr>\n";
+        $restRes[] = [
+          "pid" => $row['processid'],
+          "query" => htmlspecialchars($row[$current_query]),
+          "startTime" => $StartTime,
```

Comment:
Please use raw value `$row['query_start']` and let client decide the timezone to format it.

Reply:
Done, @GMishx